src/WebSockets/Channels/Channel.php

```diff
@@ -14,12 +14,16 @@ class Channel
     /** @var string */
     protected $channelName;
 
+    /** @var ReplicationInterface */
+    protected $pubSub;
+
     /** @var \Ratchet\ConnectionInterface[] */
     protected $subscribedConnections = [];
 
     public function __construct(string $channelName)
     {
         $this->channelName = $channelName;
+        $this->pubSub = app(ReplicationInterface::class);
```

Comment:
Hmm, I think it might be better to use DI here, which would require using the container when making Channel.

I don't have an IDE right now, not sure how complex of a change that'd be.